pkg/controller/helmrelease/helmrelease_controller.go

```diff
@@ -121,10 +120,13 @@ func (r *ReconcileHelmRelease) Reconcile(request reconcile.Request) (reconcile.R
 
 	err := r.GetClient().Get(context.TODO(), request.NamespacedName, instance)
 	if apierrors.IsNotFound(err) {
+		klog.Info("Ignorable error. Failed to find HelmRelease, most likely it has been uninstalled: ",
+			instance.GetNamespace(), "/", instance.GetName(), " ", err)
```

Comment:
since the `instance.GetNamespace(), "/", instance.GetName(),` shows a lot, it
will be better if we can have help func like this:

```go
func PrintHelper(obj metav1.Object) string{
    return fmt.Sprintf("%s/%s", obj.GetNamespace(), obj.GetName())
}
```